core/templates/pages/exploration-editor-page/translation-tab/services/translation-language.service.spec.ts

```diff
@@ -16,65 +16,61 @@
  * @fileoverview Unit test for the Translation language service.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
+// eslint-disable-next-line max-len
```

Comment:
Is this really necessary? You can long lines into 2.

Reply:
removed now thanks for the suggestions!